Notebooks/verify_source_table.ipynb

```diff
@@ -0,0 +1,323 @@
+    "\n",
+    "source_table_parquet_file = '../test_src_visit_241985.parquet'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# object_table = pd.read_parquet(object_table_parquet_file)\n",
+    "# source_table = pd.read_parquet(source_table_parquet_file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "object_table = GCRCatalogs.load_catalog(object_reader)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_table = GCRCatalogs.load_catalog(source_reader, config_overwrite=source_reader_config)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# source_table.generate_schema_yaml()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(len(object_table), len(source_table))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Match the object ID and check that we really got the right thing in the source table ID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bright_ids = pd.DataFrame(source_table.get_quantities(['sourceId', 'objectId'], native_filters=['mag_err < 0.05']))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_ids = pd.DataFrame(source_table.get_quantities(['sourceId', 'objectId', 'ra', 'dec', 'mag', 'mag_err']))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_ids['mag_err']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bright_ids = all_ids[all_ids['mag_err'] < 0.05]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(bright_ids)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.hexbin(bright_ids['mag'], bright_ids['mag_err'])\n",
+    "plt.xlabel('mag')\n",
+    "plt.ylabel('mag_err')\n",
+    "plt.xlim(14, 25)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Looking for objectId for one source\n",
+    "Let's look for all matching Sources for one object.\n",
+    "Let's just take the first entry from the bright_ids, assuming bright sources should be detected in the coadd and have an objectId.\n",
+    "\n",
+    "Basic checks:\n",
+    "  * Are these RA, Dec values agree?\n",
```

Comment:
Sorry if I missed something! What is meant by the first question?

Reply:
Fixed and grammerfied to:
"Do the RA, Dec values agree across the Source measurements?"